src/components/Posts/IndividualPost/IndividualPost.js

```diff
@@ -82,7 +83,7 @@ export const IndividualPost = () => {
               {post && `${post.updatedAt.substr(0, post.updatedAt.indexOf('T'))} `}
               by
               {` ${post && post.username}`}
-              <Chip label={post.label} size="small" color="primary" className={classes.chip} />
+              <Label label={post.label} style={classes.chip} />
```

Comment:
would it better if only pass label as props, and move chip styling to a new style.js inside the label folder?

Reply:
That is how I had it at first, but `All Post` component needs a different stylying from `Comment`s and `IndividualPost` a global styling was conflicting